modules/swagger-codegen/src/main/java/io/swagger/codegen/InlineModelResolver.java

```diff
@@ -191,6 +191,12 @@ public void flatten(Swagger swagger) {
         }
     }
 
+    private void fixStringModel(ModelImpl m) {
+        if (m.getType() != null && m.getType().equals("string") && m.getExample() != null) {
+            m.setExample(m.getExample().toString().replace("\"", ""));
```

Comment:
As @wing328 mentioned, this removes all the `"` from the example string, which likely is not what we want.
Could we restrict this to the first/last character?

Unfortunately, Swagger-models (or at least the classes/interfaces in use here) have no javadocs, so it is not really clear what should be in this `example` property.